From f403653c5ff41c6bdf7237d44fcc1752c1307271 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 1 Jan 2023 21:30:23 -0800
Subject: [PATCH] zlib: Do not undefine _FILE_OFFSET_BITS

This is required for 64bit time_t to work with glibc as it needs
_FILE_OFFSET_BITS to be defined as 64 as well.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/3rdparty/zlib/gzguts.h | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/3rdparty/zlib/gzguts.h b/src/3rdparty/zlib/gzguts.h
index 9e4368e8892..84c852fff3f 100644
--- a/src/3rdparty/zlib/gzguts.h
+++ b/src/3rdparty/zlib/gzguts.h
@@ -12,15 +12,6 @@
 #  endif
 #endif
 
-#ifdef _LARGEFILE64_SOURCE
-#  ifndef _LARGEFILE_SOURCE
-#    define _LARGEFILE_SOURCE 1
-#  endif
-#  ifdef _FILE_OFFSET_BITS
-#    undef _FILE_OFFSET_BITS
-#  endif
-#endif
-
 #include "qconfig.h"
 #if ((__GNUC__-0) * 10 + __GNUC_MINOR__-0 >= 33) && !defined(NO_VIZ) && defined(QT_VISIBILITY_AVAILABLE)
 #  define ZLIB_INTERNAL __attribute__((visibility ("hidden")))
